### ISSUE TITLE
Give advice / instructions to newcomers on the chat

### ISSUE THREAD

**Issue by icarito** [publiclab/plotsbot#72]
The following exchange from #publiclab :
```
PathFinder177 (IRC)
hey everyone
no one here ok
PathFinder177 (IRC) has left the room.
```
In a case like this, `plotsbot` should offer advice as to wait for a response or @mention people, or otherwise an appropriate bot command / behaviour.

**Comment by jywarren** [comment]
@ryzokuken - this would be a great place to use the `setTimeout` idea to make a behavoir that says something like `Hi, looks like nobody's around just now. If you have a question, you can post it a https://publiclab.org/questions, or if you need help with a software issue, you can open an issue at https://github.com/publiclab/plots2/issues/new -- thanks!`

**Comment by ryzokuken** [comment]
@jywarren sure. I will start working on that right away.

**Comment by ryzokuken** [comment]
I think we should have something of a "fallback" message triggered behavior for cases like these. Needed to talk to you regarding these. Would you be free of sometime?

**Comment by ryzokuken** [comment]
@jywarren basically, what I had in mind, was that unlike normal message triggered behaviors that have a definite keyword, we could have some special "fallback" message triggered behaviors that would *all* be called if no behaviors match. This would also be important since this is the case in which the person is not trying to talk to the bot but is trying to talk to others instead.

This one seems to be the perfect example of such a behavior.

**Comment by jywarren** [comment]
That sounds reasonable -- we can test in the testing channel to be sure it doesn't flood the channel.

**Comment by ryzokuken** [comment]
@jywarren sounds great! I am starting to work on this, will hopefully submit a PR by the end of the day.

**Comment by ryzokuken** [comment]
Resolved in #76 😄
